Jellyfin.Server.Implementations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -37,4 +37,30 @@ public static IServiceCollection AddJellyfinDbContext(this IServiceCollection se
 
         return serviceCollection;
     }
+
+    /// <summary>
+    /// Adds the <see cref="IDbContextFactory{TContext}"/> interface to the service collection with second level caching enabled.
+    /// </summary>
+    /// <param name="serviceCollection">An instance of the <see cref="IServiceCollection"/> interface.</param>
+    /// <returns>The updated service collection.</returns>
+    public static IServiceCollection AddLibraryDbContext(this IServiceCollection serviceCollection)
+    {
+        serviceCollection.AddEFSecondLevelCache(options =>
+            options.UseMemoryCacheProvider()
+                .CacheAllQueries(CacheExpirationMode.Sliding, TimeSpan.FromMinutes(10))
+                .DisableLogging(true)
+                .UseCacheKeyPrefix("EF_")
+                // Don't cache null values. Remove this optional setting if it's not necessary.
+                .SkipCachingResults(result =>
+                    result.Value is null || (result.Value is EFTableRows rows && rows.RowsCount == 0)));
```

Comment:
We don't need to add the cache services twice

I would either stick the `LibraryDbContext` part into `AddJellyfinDbContext`, or pull `AddEFSecondLevelCache` to a separate function